src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -71,33 +71,52 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			return config;
 		}
 
-		/// <summary>Bindable property for attached property <c>UseSafeArea</c>.</summary>
+		/// <summary>
+		/// Controls whether padding values are overridden with the safe area insets.
+		/// </summary>
 #if MACCATALYST
 		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), true);
 #else
 		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), false);
 #endif
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='GetUseSafeArea']/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether padding values are overridden with values that conform to the safe area on the device.
+		/// </summary>
+		/// <param name="element">The element whose safe area behavior to get.</param>
+		/// <returns>A Boolean value that tells whether padding values are overridden or not.</returns>
 		public static bool GetUseSafeArea(BindableObject element)
 		{
 			return (bool)element.GetValue(UseSafeAreaProperty);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='SetUseSafeArea'][1]/Docs/*" />
+		/// <summary>
+		/// Sets a value that controls whether padding values are overridden with the safe area insets.
+		/// </summary>
+		/// <param name="element">The element whose safe area use behavior to set.</param>
+		/// <param name="value">The new safe area inset behavior.</param>
```

Comment:
I recommend adhering to style guides for boolean parameters: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors